example/src/Product/ProductPage.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+
+import { Body, BottomSheet, Product, Screen, useTheme } from 'smartway-react-native-ui';
+
+export const ProductPage = () => {
+    const theme = useTheme();
+    const [isOpened, setOpened] = useState<booelan>(false);
+
+    return (
+        <Screen style={{ backgroundColor: theme.sw.colors.neutral[50] }}>
+            <Product
+                label="Crème dessert chocolat avec billes et chocolat avec billes et"
+                barcode="978020137962"
+                scanDate={'16:40'}
+                discount={'30%'}
+                finalPrice={'1.75€'}
+                initialPrice={'2,50€'}
+                quantity={3}
+                onPress={() => setOpened(true)}
+            />
+            <Product
+                label="Crème dessert chocolat avec billes et chocolat avec billes et"
+                barcode="978020137962"
+                scanDate={'16:40'}
+                discount={'30%'}
+                finalPrice={'1.75€'}
+                initialPrice={'2,50€'}
+                quantity={3}
+                onPress={() => setOpened(true)}
+            />
+            <Product
+                label="Crème dessert chocolat avec billes et chocolat avec billes et"
+                barcode="978020137962"
+                scanDate={'16:40'}
+                discount={'30%'}
+                finalPrice={'1.75€'}
+                initialPrice={'2,50€'}
+                quantity={3}
+                onPress={() => setOpened(true)}
+            />
+            <BottomSheet isOpened={isOpened} onClose={() => setOpened(false)}>
```

Comment:
Could you please remove the BottomSheet in this example ?
The bottom sheet is needed [in the "history" screen ](https://www.figma.com/file/yKsrpzbeh2Q02CA9HJ6tYl/SW-mobile-app---UI?node-id=127-3723&t=2kxiSS60m3YpXhvj-4), but here in the sample app it is not.

Reply:
Yes, i will remove it